crates/re_format/src/lib.rs

```diff
@@ -231,7 +231,7 @@ fn test_parse_bytes_base10() {
 }
 
 pub fn parse_bytes_base2(bytes: &str) -> Option<i64> {
```

Comment:
isn't that the function we use to parse user input in env variables? i'm a bit worried to see it being case-sensitive, that's gonna trip up a lot of people...

Reply:
I doubt anyone is specifying a memory limit in KiB, so I think this small breaking change can slip though.

As for case insensitive: I don't think we should parse `Mib` (Mebi-bits) as `MiB` (Mebi-bytes). _that_ could trip someone up.

Reply:
I added a comment about it